src/adcm_pytest_plugin/plugin.py

```diff
@@ -142,18 +142,23 @@ def parametrized_by_adcm_version(adcm_min_version=None, adcm_images=None):
 
 
 def _get_adcm_new_versions_tags(min_ver):
```

Comment:
I think will be better to split this function and write doctests with typical tags for tag parser part

Reply:
Fixed